package.json

```diff
@@ -1,41 +1,41 @@
-{
-    "name": "sonar-cnes-report-plugin",
-    "license": "GPL-3.0",
-    "version": "3.1.0",
-    "devDependencies": {
-        "@babel/core": "^7.8.7",
-        "@babel/plugin-proposal-class-properties": "^7.8.3",
-        "@babel/preset-env": "^7.8.7",
-        "@babel/preset-react": "^7.8.3",
-        "autoprefixer": "6.2.2",
-        "babel-core": "6.14.0",
-        "babel-jest": "^25.1.0",
-        "babel-loader": "^8.0.6",
-        "babel-preset-react-app": "0.2.1",
-        "css-loader": "^3.4.2",
-        "postcss-loader": "0.8.0",
-        "react": "15.6.2",
-        "react-addons-shallow-compare": "15.6.2",
-        "react-addons-test-utils": "15.6.2",
-        "react-dev-utils": "^10.2.0",
-        "react-dom": "15.6.2",
-        "react-router": "3.0.2",
-        "react-transform-hmr": "1.0.4",
-        "style-loader": "0.13.0",
-        "uglifyjs-webpack-plugin": "^2.2.0",
-        "webpack": "^4.42.0"
-    },
-    "scripts": {
-        "build": "node src/main/js/scripts/build.js"
-    },
-    "babel": {
-        "presets": [
-            "@babel/env",
-            "@babel/react"
-        ],
-        "plugins": [
-            "@babel/plugin-proposal-class-properties"
-        ]
-    },
-    "dependencies": {}
-}
+{
```

Comment:
From what I see, this file was not modified at all...
Maybe we should revert the change in Git, in case there are some hidden characters.
What do you think ?